bot/seasons/easter/egg_hunt/cog.py

```diff
@@ -0,0 +1,511 @@
+
+        if self.current_timestamp() < EggHuntSettings.start_time:
+            return
+
+        if message.channel.id not in EggHuntSettings.allowed_channels:
+            log.debug("Message not in Egg Hunt channel; ignored.")
+            return
+
+        if message.author.bot:
+            return
+
+        if random.randrange(100) <= 5:
+            await EggMessage(message, random.choice([Emoji.egg_white, Emoji.egg_blurple])).start()
+
+    @commands.group(invoke_without_command=True)
+    async def hunt(self, ctx):
+        """
+        For 48 hours, hunt down as many eggs randomly appearing as possible.
+
+        Standard Eggs
+        --------------
+        Egg React: +1pt
+        Team Bonus for Claimed Egg: +1pt
+        First React on Other Team Egg: +1pt
+        First React on Your Team Egg: +2pt
+
+        If you get first react, you will claim that egg for your team, allowing
+        your team to get the Team Bonus point, but be quick, as the egg will
+        disappear after 5 seconds of the first react.
+
+        Super Eggs
+        -----------
+        Gold Egg: 50 team pts, 5pts to first react
+        Diamond Egg: 100 team pts, 10pts to first react
+
+        Super Eggs only appear in #seasonalbot-chat so be sure to keep an eye
+        out. They stay around for 5 minutes and the team with the most reacts
+        wins the points.
+        """
+
+        await ctx.invoke(bot.get_command("help"), "hunt")
+
+    @hunt.command()
+    async def countdown(self, ctx):
+        """Show the time status of the Egg Hunt event."""
+
+        now = self.current_timestamp()
+        if now > EggHuntSettings.end_time:
+            return await ctx.send("The Hunt has ended.")
+
+        difference = EggHuntSettings.start_time - now
+        if difference < 0:
+            difference = EggHuntSettings.end_time - now
+            msg = "The Egg Hunt will end in"
+        else:
+            msg = "The Egg Hunt will start in"
+
+        hours, r = divmod(difference, 3600)
+        minutes, r = divmod(r, 60)
+        await ctx.send(f"{msg} {hours:.0f}hrs, {minutes:.0f}mins & {r:.0f}secs")
+
+    @hunt.command()
+    async def leaderboard(self, ctx):
+        """Show the Egg Hunt Leaderboards."""
+
+        db = sqlite3.connect(DB_PATH)
+        c = db.cursor()
+        c.execute(f"SELECT *, RANK() OVER(ORDER BY score DESC) AS rank FROM user_scores LIMIT 10")
+        user_result = c.fetchall()
+        c.execute(f"SELECT * FROM team_scores ORDER BY team_score DESC")
+        team_result = c.fetchall()
+        db.close()
+        output = []
+        scr_len = max(len(str(r[2])) for r in user_result)
+        for user_id, team, score, rank in user_result:
+            user = GUILD.get_member(user_id) or user_id
+            team = team.capitalize()
+            score = f"{score}pts"
+            output.append(f"{rank:>2}. {score:>{scr_len+3}} - {user} ({team})")
+        user_board = '\n'.join(output)
```

Comment:
might want to use double quotes here as well for the sake of consistency

Reply:
Will do, cheers.